secp-api/src/main/java/org/bitcoinj/secp/api/P256k1PubKey.java

```diff
@@ -31,23 +31,16 @@ default String getAlgorithm() {
 
     @Override
     default String getFormat() {
-        return "Uncompressed SEC";
+        return "Compressed SEC";
     }
 
     /**
-     * Return key in primary encoded format (uncompressed for now)
-     * @return public key in uncompressed format
+     * Return key in primary encoded format (compressed)
+     * @return public key in compressed format
      */
     @Override
     default byte[] getEncoded() {
-        ECPoint point = getW();
-        byte[] x = integerTo32Bytes(point.getAffineX());
-        byte[] y = integerTo32Bytes(point.getAffineY());
-        byte[] encoded = new byte[65];
-        encoded[0] = 0x04;
-        System.arraycopy(x, 0, encoded, 1, 32);
-        System.arraycopy(y, 0, encoded, 33, 32);
-        return encoded;
+        return getUncompressed();
```

Comment:
This seems to be contrary to the Javadoc description?

Reply:
Good catch!

Apparently, I messed that up when I converted from one commit to two. How embarrassing. Even more embarrassing is that I don't have a test to catch the mistake. I do plan on adding tests!

Reply:
Fixed and force-pushed.